index.html

```diff
@@ -53,90 +50,159 @@ <h3>I'm a Software Developer</h3>
+        <div class="img1">
+            <img src="images/offenbach.png" alt="">
         </div>
-        <div class="items description1">
-            <h3 class="title2">Multi-Post Stories</h3>
-            <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="description">
+            <h3>Offenbach GP</h3>
+            <p>A fully online opera-singing competition, where the audience can comment and applaud, and where the jury comments in full view of the audience.  Automated application process, which includes video and biog upload, as well as an automatically generated contract in the end.</p>
         </div>
-        <ul class="items lang">
+        <ul>
             <li>html</li>
             <li>css</li>
             <li>php</li>
             <li>mysql</li>
             <li>javascript</li>
             
         </ul>
-        <div class="items buttontypeA">
-        <a href="#">See Project</a>
+        <div class="buttontypeA">
+        <a href="https://www.offenbachgp.com/" rel=”noopener”>See Project</a>
         </div>
-        <div class="items site1">
-            <div class="description2">
-                <div class="title3">Multi-Post Stories</div>
-                <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-            </div>
-            <ul class="items lang">
-                <li>html</li>
-                <li>css</li>
-                <li>php</li>
-                <li>mysql</li>
-                <li>javascript</li>
-                
-            </ul>
-            <div class="items buttontypeB">
-                <a href="#">See Project</a>
-            </div>
+    </div><!--container end-->
+    
+    <div class="container2">
+        <div class="site site1">
+         <a href="https://spoiledeggs.eu5.org/" rel=”noopener”><div class="greyout">
+                <div class="description">
+                    <div class="title">Spoiled Eggs</div>
+                    <p class="text">Spoiled Eggs was the original template for 2 projects and those two projects were used to promote Spoiled Eggs.  It's a review site for opera, where people can rage click "applause" or "throwing eggs" and of course comment.</p>
+                </div>
+                <ul class="items lang">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>php</li>
+                    <li>mysql</li>
+                    <li>javascript</li>
+                    
+                </ul>
+            </div><!--greyout end-->
+            <div class="buttondecor">
+                <p>See Project</p>
+            </div></a>
```

Comment:
- [ ] I can see that your cards are not really positioned to the center of your page the right padding is larger than the left look :point_down: 

![022](https://user-images.githubusercontent.com/57562869/149396637-04c7b3b4-c67c-426a-9016-c68fe66a25da.png)

- Kindly make your cards to be to the center of your page